torch/ao/quantization/qconfig.py

```diff
@@ -249,6 +258,12 @@ def get_default_qat_qconfig(backend='fbgemm', version=1):
                                                                                  quant_max=255,
                                                                                  reduce_range=False),
                               weight=default_fused_wt_fake_quant)
+        elif backend == 'onednn':
+            qconfig = QConfig(activation=FusedMovingAvgObsFakeQuantize.with_args(observer=MovingAverageMinMaxObserver,
+                                                                                 quant_min=0,
+                                                                                 quant_max=255,
+                                                                                 reduce_range=False),
```

Comment:
should the reduce range be True here? or we can set qunat_min/quant_max directly, like https://github.com/pytorch/pytorch/pull/71027/files since we are moving away from using reduce_range in configurations